watertap/data/techno_economic/ion_exchange.yaml

```diff
@@ -91,3 +100,38 @@ cation_exchange:
     capital_d_parameter:
       value: 4.09643E-05
       units: dimensionless
+clinoptilolite: #AMO subtype
+  recovery_frac_mass_H2O:
+    value: 0.973
+    units: dimensionless
+  default_removal_frac_mass_solute:
+    value: 0
+    units: dimensionless
+  removal_frac_mass_solute:
+    ammonium_as_nitrogen:
+      value: 0.999
+      units: dimensionless
+      constituent_longform: Total Dissolved Solids (TDS)
+  NaCl_dose:  # keep from default
+    value: 0    # unsure if applicable
+    units: kg/m^3
+  resin_replacement:  # keep from default
```

Comment:
Does the same apply for capital costs?

Reply:
It should. The load parameter method starts by getting the default values, and then overloading the ones changed by the sub-type.

Reply:
And actually- in this case - these aren't repeated values. I set these to 0 since NaCl_dose may not be necessary for the subtype, and resin_replacement is not known yet so I set that to 0 as well temporarily. Using the defaults would assign values that I don't want to include just yet.